README.md

```diff
@@ -117,7 +117,9 @@ pip install git+https://github.com/ml6team/fondant.git
 
 ### Deploying Fondant
 
-TODO
+Fondant is flexible and works by leveraging [kubeflow pipelines](https://www.kubeflow.org/docs/components/pipelines/v1/introduction/) on any kubernetes cluster. All fondant needs is an url pointing to the kubeflow pipeline host and a Object Storage provider (S3, GCS, etc) to store data produced in the pipeline between steps.
+
+We have compiled some references and created some scripts to [get you started](docs/infrastructure.md) with setting up the required infrastructure.
```

Comment:
```suggestion
We have compiled some references and created some scripts to [get you started](https://fondant.readthedocs.io/en/latest/manifest/) with setting up the required infrastructure.
```

Reply:
I guess we still want to ref infrastructure and not manifest ?

Reply:
Woops, yes :)

Reply:
I would expect it to work based on:
https://www.mkdocs.org/user-guide/writing-your-docs/#linking-to-pages

Unless I'm misunderstanding

Reply:
Just tested locally with `mkdocs serve` and just referencing the .md file works because mkdocs will convert it to the right url.

Reply:
Ok, it didn't work on the test build on readthedocs.

Reply:
because I used `docs/infrastructure.md` but it should just be `infrastructure.md`

Reply:
well I did not think about that. I'll make the change